topcoffea/modules/selection.py

```diff
@@ -175,6 +174,7 @@ def trgPassNoOverlap(events,is_data,dataset,year):
 
     # In case of data, check if events overlap with other datasets
     if is_data:
+        trg_passes = passesTrgInLst(events,dataset_dict[year][dataset])
```

Comment:
@sscruz thanks for catching this!

Just to be explicit about what the issue is so that I'm not missing something. The reason for this logic change is for the edge case where an event from dataset "A" doesn't pass any of the analysis triggers for dataset "A", but _does_ pass an analysis trigger from dataset "B". This means that it could be double counted b/c the overlap removal won't remove the event when it showed up in dataset "A" (b/c it didn't pass any of the analysis level triggers for that dataset) while still allowing the event to be included when it shows up in dataset "B". Is that correct?

I'm pretty sure this should be a pretty small effect. When I tested the original implementation by shuffling the order of the overlap removal, I saw _almost_ no differences in the distributions. Since the differences were so small I didn't investigate it any further at the time, but I'm guessing this is the reason for those differences.

In any case, this is a good catch and I am fine with this change!